manage-data/data-store/index-basics.md

```diff
@@ -15,10 +15,152 @@ mapped_urls:
+
+::::{tip} 
+A closely related concept is a [data stream](/manage-data/data-store/index-types/data-streams.md). This index abstraction is optimized for append-only timestamped data, and is made up of hidden, auto-generated backing indices. If you’re working with timestamped data, we recommend the [Elastic Observability](https://www.elastic.co/guide/en/observability/current) solution for additional tools and optimized content.
+::::
+
+## Index components
+
+An index is made up of the following components.
+
+### Documents [elasticsearch-intro-documents-fields] 
+
+{{es}} serializes and stores data in the form of JSON documents. A document is a set of fields, which are key-value pairs that contain your data. Each document has a unique ID, which you can create or have {{es}} auto-generate.
+
+A simple {{es}} document might look like this:
+
+```js
+{
+  "_index": "my-first-elasticsearch-index",
+  "_id": "DyFpo5EBxE8fzbb95DOa",
+  "_version": 1,
+  "_seq_no": 0,
+  "_primary_term": 1,
+  "found": true,
+  "_source": {
+    "email": "john@smith.com",
+    "first_name": "John",
+    "last_name": "Smith",
+    "info": {
+      "bio": "Eco-warrior and defender of the weak",
+      "age": 25,
+      "interests": [
+        "dolphins",
+        "whales"
+      ]
+    },
+    "join_date": "2024/05/01"
+  }
+}
+```
+
+### Metadata fields [elasticsearch-intro-documents-fields-data-metadata] 
+
+An indexed document contains data and metadata. [Metadata fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html) are system fields that store information about the documents. In {{es}}, metadata fields are prefixed with an underscore. For example, the following fields are metadata fields:
+
+* `_index`: The name of the index where the document is stored.
+* `_id`: The document’s ID. IDs must be unique per index.
+
+
+### Mappings and data types [elasticsearch-intro-documents-fields-mappings] 
+
+Each index has a [mapping](/manage-data/data-store/mapping.md) or schema for how the fields in your documents are indexed. A mapping defines the [data type](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html) for each field, how the field should be indexed, and how it should be stored.
+
+## Index management
+
+Elastic's index management features are an easy, convenient way to manage your cluster's indices, data streams, index templates, and enrich policies. Practicing good index management ensures your data is stored correctly and in the most cost-effective way possible.
+
+Go to **{{project-settings}} → {{manage-app}} → {{index-manage-app}}**.
+
+### Manage indices
+
+Investigate your indices and perform operations from the **Indices** view.
+
+:::{image} /images/serverless-index-management-indices.png
```

Comment:
The images in this PR aren't showing up for me in the preview... I'm not sure, but this might help.

```suggestion
:::{image} ../../images/serverless-index-management-indices.png
```

Reply:
Ah I had to hard refresh to get the page built with the latest docs-builder update (including the redesign). After that I can see the images. 👍